src/main/scala/firrtl/ir/IR.scala

```diff
@@ -285,6 +285,13 @@ case class DefInstance(info: Info, name: String, module: String) extends Stateme
   def foreachString(f: String => Unit): Unit = f(name)
   def foreachInfo(f: Info => Unit): Unit = f(info)
 }
+
+object ReadUnderWrite extends Enumeration {
+  val UNDEFINED = Value("undefined")
+  val OLD = Value("old")
+  val NEW = Value("new")
```

Comment:
Should these be CamelCase?

See e.g., [PrimOps](https://github.com/freechipsproject/firrtl/blob/f08f8dbb3c480220f92923a7f3242fcbb644b65e/src/main/scala/firrtl/PrimOps.scala) which uses CamelCase for enum like constans.

Reply:
Yeah, I agree. I was copying the pattern from `Gender`, which used all-caps, but it looks like the switch to `Flow` has eliminated the all-caps.